{"uid":"1a1e5a267ef2590","name":"test_list_address_nullifier_const_11_28","fullName":"tests.test_checklist.TestPromotionChecklist#test_list_address_nullifier_const_11_28","historyId":"f80c5f439b459290223bced1d24c1a8c","time":{"start":1739795575413,"stop":1739795737112,"duration":161699},"status":"failed","statusMessage":"AssertionError: False is not true : {'list_address-properties-address': True, 'list_address-properties-state_prov': False, 'list_address-properties-postal_code': False, 'list_address-properties-street_name': False, 'list_address-properties-street_number': False, 'list_address-properties-unit_number': False, 'list_address-properties-street_suffix': False, 'list_address-properties-street_post_dir': False, 'list_address-properties-street_direction': False}","statusTrace":"tests/test_checklist.py:91: in test_list_address_nullifier_const\n    raise e  # Re-raise to ensure the test fails\ntests/test_checklist.py:86: in test_list_address_nullifier_const\n    self.assertTrue(all(actual), result)\nE   AssertionError: False is not true : {'list_address-properties-address': True, 'list_address-properties-state_prov': False, 'list_address-properties-postal_code': False, 'list_address-properties-street_name': False, 'list_address-properties-street_number': False, 'list_address-properties-unit_number': False, 'list_address-properties-street_suffix': False, 'list_address-properties-street_post_dir': False, 'list_address-properties-street_direction': False}","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist","time":{"start":1739788254308,"stop":1739788254308,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: False is not true : {'list_address-properties-address': True, 'list_address-properties-state_prov': False, 'list_address-properties-postal_code': False, 'list_address-properties-street_name': False, 'list_address-properties-street_number': False, 'list_address-properties-unit_number': False, 'list_address-properties-street_suffix': False, 'list_address-properties-street_post_dir': False, 'list_address-properties-street_direction': False}","statusTrace":"tests/test_checklist.py:91: in test_list_address_nullifier_const\n    raise e  # Re-raise to ensure the test fails\ntests/test_checklist.py:86: in test_list_address_nullifier_const\n    self.assertTrue(all(actual), result)\nE   AssertionError: False is not true : {'list_address-properties-address': True, 'list_address-properties-state_prov': False, 'list_address-properties-postal_code': False, 'list_address-properties-street_name': False, 'list_address-properties-street_number': False, 'list_address-properties-unit_number': False, 'list_address-properties-street_suffix': False, 'list_address-properties-street_post_dir': False, 'list_address-properties-street_direction': False}","steps":[{"name":"Click authorisation button","time":{"start":1739795576454,"stop":1739795583920,"duration":7466},"status":"passed","steps":[{"name":"Get authorisation button","time":{"start":1739795576454,"stop":1739795576464,"duration":10},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Source select","time":{"start":1739795584455,"stop":1739795584575,"duration":120},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"source","value":"'28'"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795585959,"stop":1739795586058,"duration":99},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"1"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795586090,"stop":1739795594044,"duration":7954},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"\nMetadata = PROPERTY Failed ‚ùå \n","time":{"start":1739795594216,"stop":1739795594216,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795594235,"stop":1739795594363,"duration":128},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"2"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795594401,"stop":1739795604443,"duration":10042},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class CUSTOMPROPERTY is not mapped","time":{"start":1739795604444,"stop":1739795604444,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795604444,"stop":1739795604564,"duration":120},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"3"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795604599,"stop":1739795614636,"duration":10037},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class DATASYSTEM is not mapped","time":{"start":1739795614636,"stop":1739795614637,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795614637,"stop":1739795614754,"duration":117},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"4"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795614779,"stop":1739795624836,"duration":10057},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class ENUMERATION is not mapped","time":{"start":1739795624837,"stop":1739795624837,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795624837,"stop":1739795624960,"duration":123},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795624983,"stop":1739795635015,"duration":10032},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYROOMS is not mapped","time":{"start":1739795635015,"stop":1739795635016,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795635016,"stop":1739795635160,"duration":144},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"6"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795635189,"stop":1739795645236,"duration":10047},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYUNITTYPES is not mapped","time":{"start":1739795645237,"stop":1739795645237,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795645237,"stop":1739795645392,"duration":155},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"7"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795645428,"stop":1739795655444,"duration":10016},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYBUILDING is not mapped","time":{"start":1739795655445,"stop":1739795655445,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795655445,"stop":1739795655583,"duration":138},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"8"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795655612,"stop":1739795665668,"duration":10056},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLASTMARKETSALE is not mapped","time":{"start":1739795665669,"stop":1739795665669,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795665669,"stop":1739795665806,"duration":137},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"9"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795665831,"stop":1739795675888,"duration":10057},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLOCATION is not mapped","time":{"start":1739795675889,"stop":1739795675889,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795675889,"stop":1739795676021,"duration":132},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"10"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795676044,"stop":1739795686074,"duration":10030},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIP is not mapped","time":{"start":1739795686075,"stop":1739795686075,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795686075,"stop":1739795686211,"duration":136},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"11"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795686232,"stop":1739795696267,"duration":10035},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIPTRANSFER is not mapped","time":{"start":1739795696268,"stop":1739795696268,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795696268,"stop":1739795696408,"duration":140},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"12"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795696429,"stop":1739795706446,"duration":10017},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYPRIORSALE is not mapped","time":{"start":1739795706447,"stop":1739795706447,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795706447,"stop":1739795706591,"duration":144},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"13"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795706612,"stop":1739795716652,"duration":10040},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSEARCH is not mapped","time":{"start":1739795716653,"stop":1739795716653,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795716653,"stop":1739795716797,"duration":144},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"14"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795716819,"stop":1739795726835,"duration":10016},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSITE is not mapped","time":{"start":1739795726836,"stop":1739795726836,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739795726836,"stop":1739795726987,"duration":151},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"15"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739795727007,"stop":1739795737047,"duration":10040},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55ea28ab209a <unknown>\n#1 0x55ea285798b0 <unknown>\n#2 0x55ea285cab99 <unknown>\n#3 0x55ea285cad21 <unknown>\n#4 0x55ea28619854 <unknown>\n#5 0x55ea285f09ad <unknown>\n#6 0x55ea28616b8b <unknown>\n#7 0x55ea285f0753 <unknown>\n#8 0x55ea285bc38e <unknown>\n#9 0x55ea285bdb51 <unknown>\n#10 0x55ea28a7b76b <unknown>\n#11 0x55ea28a7f6f2 <unknown>\n#12 0x55ea28a678fc <unknown>\n#13 0x55ea28a802e4 <unknown>\n#14 0x55ea28a4b9cf <unknown>\n#15 0x55ea28aa0cd8 <unknown>\n#16 0x55ea28aa0eb6 <unknown>\n#17 0x55ea28ab0f16 <unknown>\n#18 0x7fc71289caa4 <unknown>\n#19 0x7fc712929c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYTAXASSESSMENT is not mapped","time":{"start":1739795737048,"stop":1739795737048,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":48,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist::0","time":{"start":1739801612961,"stop":1739801612961,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.filterwarnings('ignore::DeprecationWarning')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_checklist"},{"name":"subSuite","value":"TestPromotionChecklist"},{"name":"host","value":"fv-az1043-579"},{"name":"thread","value":"3351-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_checklist"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":["@pytest.mark.filterwarnings('ignore::DeprecationWarning')"]},"source":"1a1e5a267ef2590.json","parameterValues":[]}